a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -986,31 +986,23 @@ <h4>Examples</h4>
 						<ul>
 							<li><span data-localization-id="hazards-none" data-localization-mode="descriptive">No hazards</span></li>
 							<li><span data-localization-id="hazards-flashing" data-localization-mode="descriptive">The
-									publication contains </span>
-								<span data-localization-id="hazards-sound" data-localization-mode="descriptive">rapidly
-									changing lights,</span>
-								<span data-localization-id="hazards-motion" data-localization-mode="descriptive">visual
-									stimuli or simulated movements, </span>
-								<span data-localization-id="hasards-explanatory" data-localization-mode="descriptive"
-									>which can cause discomfort, distraction, photosensitive seizures, or motion
-									sickness</span></li>
-							<li><span data-localization-id="hazards-unknown" data-localization-mode="descriptive">The
-									presence of hazards is unknown</span></li>
+									publication contains rapidly changing lights which can cause photosensitive seizures</span></li>
```

Comment:
I'm not sure the focus on light is the best description. It sounds like we're only concerned with strobe and other lighting effects, for example, when any flashing or blinking content (animated gifs, css animations, etc.) can be hazardous.

It might be better to generalize this to: "The publication contains flashing content which can cause photosensitive seizures" (Similarly for the compact statement.)